okhttp/src/main/java/com/squareup/okhttp/CertificatePinner.java

```diff
@@ -123,17 +123,45 @@ private CertificatePinner(Builder builder) {
    * pinned for {@code hostname}. OkHttp calls this after a successful TLS
    * handshake, but before the connection is used.
    *
+   * @return {@code true} if hostname has pinned certificates and check passed,
```

Comment:
After all discussion, I think we should probably skip the boolean here. We don't need it, and it's a bit confusing because there's three states, but most users would expect only two. (Okay, not okay).